package.json

```diff
@@ -15,16 +15,16 @@
   "license": "MIT",
   "author": "Isaac J. Lee",
   "type": "module",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
```

Comment:
I'm unsure how I should update `main`, `types`, and `bin`.

Will try updating the paths based on what `npx ember-codemod-pod-to-octane@1.2.0` installed on my local machine.

<img width="900" alt="" src="https://github.com/ijlee2/ember-codemod-pod-to-octane/assets/16869656/b705e485-1e19-4718-9c08-605f8c83cfbd">